scripts/generator/graphql_generator/csharp/SDK/ShippingMethod.cs.erb

```diff
@@ -16,10 +19,30 @@ namespace <%= namespace %>.SDK {
             Detail = detail;
         }
 
+        public static ShippingMethod CreateFromJson(Dictionary<string, object> dataJSON) {
+            string label = null;
+            if (dataJSON.ContainsKey(LABEL)) {
+                label = dataJSON[LABEL].ToString();
+            }
+            string amount = null;
+            if (dataJSON.ContainsKey(AMOUNT)) {
+                amount = dataJSON[AMOUNT].ToString();
+            }
+            string identifier = null;
+            if (dataJSON.ContainsKey(IDENTIFIER)) {
+                identifier = dataJSON[IDENTIFIER].ToString();
+            }
+            string detail = null;
+            if (dataJSON.ContainsKey(DETAIL)) {
+                amount = dataJSON[DETAIL].ToString();
```

Comment:
Copy/paste error. Should be assigning to detail not amount!

Reply:
Good catch! https://github.com/Shopify/unity-buy-sdk/pull/355/commits/3883d129eaf86fe8b1199fc3984352e95bb9055c